cmd/podman/main.go

```diff
@@ -62,6 +62,7 @@ func main() {
 
 	app.Commands = []cli.Command{
 		attachCommand,
+		checkpointCommand,
```

Comment:
I think these are missing?

Reply:
Yes, this is still wrong. Need to fix it. Thanks for all the reviews.